src/plugins/kibana_react/public/table_list_view/table_list_view.tsx

```diff
@@ -67,6 +67,8 @@ export interface TableListViewProps {
   tableListTitle: string;
   toastNotifications: ToastsStart;
   uiSettings: IUiSettingsClient;
+  ariaDescribedby?: string;
```

Comment:
`TableListView` already takes in quite a few textual inputs (`entityName`, `entityNamePlural`, `tableListTitle`).
Could those be used for this?

If the fact they are translated is an issue, I would still pass in a single input, instead of the two added here (`ariaDescribedby`, `headingId`)?.

Also, `TableListView` is used in only two other places. Would you mind adding this new attribute (if eventually needed) there as well?

Reply:
The reason I introduced new properties is that in some cases the same id has to be used outside of the component. I'm fine with just using a single property - it takes away some flexibility in using the component, but for the current usage it's not relevant.